packages/sources/alphachain/src/config/limits.json

```diff
@@ -0,0 +1,13 @@
+{
+  "alphachain": {
```

Comment:
Can we make it possible to not have to define the EA name here? Since it's split up into separate files this should be possible, right?